lms/djangoapps/courseware/features/problems.feature

```diff
@@ -223,10 +223,9 @@ Feature: LMS.Answer problems
         Given I am viewing a "<ProblemType>" problem
         When I answer a "<ProblemType>" problem "<InitialCorrectness>ly"
         Then my "<ProblemType>" answer is marked "<InitialCorrectness>"
-        And I input an answer on a "<ProblemType>" problem "<OtherCorrectness>ly"
+        And I reset the problem
```

Comment:
Reviewers: it took me a long time to be sure of this, but I believe that the Lettuce test was written incorrectly as it didn't match the comment. Before it wasn't testing the reset behavior because it did no testing after it performed the reset. Instead if was verifying that the answer was not marked when providing an incorrect answer, which isn't true.

An interesting question is why this test used to pass before this upgrade. I'll investigate that tomorrow. Either way, I believe that the test as now written is verifying what the comment (and test name) suggests that it is.